src/pages/admin/homegallery/index.tsx

```diff
@@ -1,16 +1,25 @@
 import { authorizedPageSSP } from "@components/auth/auth-redirect";
 import HomeComponent from "@components/pages/admin/homegallery";
 import { Role } from "@interfaces/custom";
+import { axGetAdminHomeInfo } from "@services/utility.service";
 import React from "react";
 
-import { getServerSideProps as homePageData } from "../../index";
-
 const HomePage = ({ homeInfo }) => <HomeComponent homeInfo={homeInfo} />;
 
 export const getServerSideProps = async (ctx) => {
   const redirect = authorizedPageSSP([Role.Admin], ctx);
   if (redirect) return redirect;
-  return homePageData(ctx);
+
+  const { data: homeInfo } = await axGetAdminHomeInfo(ctx);
+
+  return {
+    props: {
+      homeInfo: {
+        ...homeInfo,
+        gallerySlider: homeInfo.gallerySlider?.sort((a, b) => a.displayOrder - b.displayOrder)
```

Comment:
is sorting still required here? I think since order matters now back-end should just query it by `displayOrder` and send it!!